File: Backend-Java/src/main/java/org/example/Infrastructure/dto/commentDTO/CommentItemDTO.java
package org.example.Infrastructure.dto.commentDTO;

import lombok.Data;
import org.example.Infrastructure.dto.accountDTO.UserItemDTO;
import org.springframework.web.multipart.MultipartFile;

import java.util.List;
@Data
public class CommentItemDTO {
    private int id;
    private String title;
    private String message;
    private int stars;
    private String dateCreated;
    private String dateUpdated;
    private int likes;
    private int dislikes;
    private int userId;
    private String userName;
    private UserItemDTO user;
    private Integer productId;
    private List<CommentImageItemDTO> images;
}

File: Backend-C#/Infrastructure.MyShop/Models/DTO/CommentDTO/CommentItemDTO.cs
using Infrastructure.MyShop.Models.DTO.AccountDTO;
using Infrastructure.MyShop.Models.DTO.ProductDTO;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Infrastructure.MyShop.Models.DTO.CommentDTO
{
    public class CommentItemDTO
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Message { get; set; }
        public int Stars { get; set; }
        public string DateCreated { get; set; }
        public string DateUpdated { get; set; }
        public int Likes { get; set; }
        public int Dislikes { get; set; }
        public long? UserId { get; set; }
        public string UserName { get; set; }
        public int? ProductId { get; set; }
        public UserItemDTO User { get; set; }
        public List<CommentImageItemDTO> Images { get; set; }
    }
}
